.gitignore

```
@@ -1 +1,2 @@
-.env
\ No newline at end of file
+.env
+final_env
```

@camlebrun Pensez svp à rajouter les dossiers __pycache__ dans votre .gitignore: 
```suggestion
final_env
__pycache__/
```
FYI pour que ça prenne effet, il faut supprimer tout les dossiers pycache, commit ce changement, puis commit le .gitignore pour ignorer les fichiers pycache :)